apps/els_dap/src/els_dap.erl

```diff
@@ -14,18 +14,28 @@
 %%==============================================================================
 -include_lib("kernel/include/logger.hrl").
 
--define(APP, erlang_ls).
+-define(APP, els_dap).
+-define(FRAMEWORK, els_framework).
 -define(DEFAULT_LOGGING_LEVEL, "debug").
 
 -spec main([any()]) -> ok.
 main(Args) ->
   application:load(getopt),
   application:load(?APP),
+  application:load(?FRAMEWORK),
+  Config1 = application:get_all_env(?FRAMEWORK),
```

Comment:
Leftovers from debugging? Or do you actually want to log these?

Reply:
yeah, this needs cleanup